### PR TITLE
Remove support for Ansible 2.8

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ We actively test the modules against the latest stable Foreman release and the m
...
 
-The supported Ansible versions are aligned with currently maintained Ansible versions that support Collections (2.8+).
-You can find the list of maintained Ansible versions [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status).
+The supported Ansible versions are aligned with currently maintained Ansible versions that support Collections (2.9+).
+You can find the list of maintained Ansible versions [here](https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html).
 
 ### Supported Python Versions
 
-Starting with Ansible 2.7, Ansible only supports Python 2.7 and 3.5 (and higher). These are also the only Python versions we develop and test the modules against.
+Ansible only supports Python 2.7 and 3.5 (and higher). These are also the only Python versions we develop and test the modules against.
```
Comment (evgeni) [inline README.md:25]: Given we dropped CI for 3.5 in 30777fd50e78f9751d48faa5c979520ccc216e55, we should bump this to 2.7/3.6+, but I am yet to find an upstream Ansible doc which says which Python versions they support.

Comment (evgeni) [reply README.md:25]: On the other hand, 3.5 is still supported by Ansible (pre 2.13), so maybe we should add it back?

Comment (evgeni) [reply README.md:25]: I've pushed a commit adding 3.5 back to the CI matrix.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -50,18 +51,13 @@ jobs:
         run: sudo apt-get install -y libyaml-dev
       - name: Install Ansible
         run: pip install --upgrade git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}
-      - name: Set Environment to use mazer
-        run: |
-          echo "COLLECTION_COMMAND=mazer" >> $GITHUB_ENV
-          pip install --upgrade mazer
-        if: matrix.ansible == 'stable-2.8'
```
Comment (mdellweg) [inline .github/workflows/main.yml:57]: Finally :checkered_flag:

**Review by mdellweg — APPROVED** [review]
LGTM

**Pull request merged**